crates/sui-types/src/executable_transaction.rs

```diff
@@ -42,21 +41,16 @@ impl CertificateProof {
         Self::SystemTransaction(epoch)
     }
 
-    pub fn new_from_consensus(
-        epoch: EpochId,
-        round: Round,
-        authority: AuthorityIndex,
-        transaction_index: TransactionIndex,
-    ) -> Self {
-        Self::Consensus(epoch, round, authority, transaction_index)
+    pub fn new_from_consensus(epoch: EpochId) -> Self {
```

Comment:
Do you mind explaining how its safe to change these `CertificateProof` without protocol upgrade? Is the consensus proof only used locally right now

Reply:
Yes this variant is not used anywhere right now. Otherwise we will have to create a new variant.